piwik/etc/confd/templates/virtual-host.conf.tmpl

```diff
@@ -0,0 +1,17 @@
+<VirtualHost *:80>
```

Comment:
This file may not be needed, as the templates in https://github.com/continuouspipe/dockerfiles/tree/master/php-apache/etc/confd/templates/apache should handle this.